internal/ethapi/api.go

```diff
@@ -2017,7 +2017,7 @@ func runMEVM(ctx context.Context, b Backend, state *state.StateDB, header *types
 	}
 
 	if result.Failed() {
-		return nil, nil, nil, fmt.Errorf("%w: %s", result.Err, hexutil.Encode(result.Revert()))
+		return nil, result, storeFinalize, nil
```

Comment:
`result.Failed()` indicates an error during the execution but **it should not be treated** as a fatal error for the call. Functions that use `runMEVM` need to differentiate between fatal errors and execution errors.

Reply:
Ditto.